src/main/java/uk/gov/pay/api/service/SearchRefundsService.java

```diff
@@ -17,12 +16,15 @@
 import java.util.Optional;
 
 import static uk.gov.pay.api.model.RefundError.Code.SEARCH_REFUNDS_DIRECT_DEBIT_ERROR;
+import static uk.gov.pay.api.validation.RefundSearchValidator.*;
```

Comment:
We don't use wildcards usually

Reply:
sorry, I've setup intellij to not do this but it keeps collapsing imports 😢